microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Get a Microsoft Store billing service by using the Digital Goods API
+description: Use the Digital Goods API to get a Microsoft Store billing service.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: pwa
+ms.date: 11/21/2024
+---
+# Get a Microsoft Store billing service by using the Digital Goods API
+<!-- todo: improve title (in 3+ spots) -->
+<!-- todo: improve the description: field above; sync w/ Sentence 1; develop clear wording for both spots -->
```

Comment:
improve the **description:** field above; sync w/ Sentence 1; develop clear wording for both spots
@runyuany

Reply:
result:
description: How to provide in-app purchases in a Progressive Web App by using the Digital Goods API and the Payment Request API.